src/main/java/com/faforever/client/chat/PrivateChatTabController.java

```diff
@@ -51,9 +98,118 @@ public void setReceiver(String username) {
     super.setReceiver(username);
     privateChatTabRoot.setId(username);
     privateChatTabRoot.setText(username);
+
+    //Load receiver information for detail pane
```

Comment:
We don't use such comments. If they seem necessary, the code should be extracted to a method with a proper name instead.